packages/bot-web-ui/src/components/quick-strategy/form-wrappers/mobile-form-wrapper.tsx

```diff
@@ -58,51 +59,84 @@ const MobileFormWrapper: React.FC<TMobileFormWrapper> = observer(({ children })
+                                            })}
                                             key={tab.value}
                                             onClick={() => setActiveTab(tab.value)}
                                         >
-                                            <Text size='xs' weight={active ? 'bold' : 'normal'}>
+                                            <Text size='xs' weight={active ? 'bold' : 'lighter'}>
                                                 {tab.label}
                                             </Text>
                                         </span>
                                     );
                                 })}
                             </div>
-                        </div> */}
-                        <div className='qs__body__content__form'>{children}</div>
+                        </div>
+                        {active_tab === 'TRADE_PARAMETERS' ? (
+                            <>
+                                <div className='qs__body__content__description'>
+                                    <div>
+                                        <Text size='xxs'>{strategy.description}</Text>
+                                    </div>
+                                </div>
+                                <div className='qs__body__content__form'>{children}</div>
+                            </>
+                        ) : (
+                            <div className='qs__body__content__description'>
+                                <div>
+                                    {strategy?.long_description?.map((data, index) => {
+                                        return (
+                                            <div key={index}>
+                                                <div className='long_description__title'>
+                                                    <Text size='xxs' weight='bold'>
+                                                        {data.title}
+                                                    </Text>
+                                                </div>
+                                                <div className='long_description__content'>
+                                                    <Text size='xxs'>{data.content}</Text>
+                                                </div>
+                                                {data.image && (
+                                                    <div>
+                                                        <img className='long_description__image' src={data.image} />
+                                                    </div>
+                                                )}
+                                            </div>
+                                        );
+                                    })}
+                                </div>
+                            </div>
+                        )}
```

Comment:
could we separate this code block out into its own component? seems like its the same as the one in desktop-form-wrapper with some minor changes like the className for the img, but could be passed as a prop or to the new component

Reply:
Yeah sure we can. Done ✅